packages/passport/sdk/src/zkEvm/zkEvmProvider.ts

```diff
@@ -123,7 +123,9 @@ export class ZkEvmProvider implements Provider {
           this.#callSessionActivity(user.zkEvm.ethAddress);
         }
       }
-    }).catch(); // User does not exist, don't initialise an eth signer
+    }).catch(() => {
+      // User does not exist, don't initialise an eth signer
+    });
```

Comment:
This was where the error was not being caught - catch requires a function to handle the error